perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/MailManagerImpl.java

```diff
@@ -1062,6 +1064,43 @@ private String substituteCommonStringsForInvite(Vo vo, Group group, User user, S
 			mailText = mailText.replace(FIELD_MAIL_FOOTER, (footer != null) ? footer : EMPTY_STRING);
 		}
 
+		// HTML mail footer
+		if (mailText.contains(FIELD_HTML_MAIL_FOOTER)) {
+			String footer = EMPTY_STRING;
+			// get proper value from attribute
+			try {
+				Attribute attribute;
+				if (group != null) {
+					if (isAlternativePlainText) {
```

Comment:
if you'd extract the content of the `if (mailText.contains(FIELD_MAIL_FOOTER)) {` block to private method, we could reuse it if `isAlternativePlainText == true` to avoid code duplication, same fot the other method